.github/workflows/preview-environment-deployment.yml

```diff
@@ -66,7 +66,7 @@ jobs:
       #     push: false
       #     # Update tags and ECR Repository
       #     tags: |
-      #       ${{ steps.login-ecr.outputs.registry }}/dsva/preview-environment/vets-website:${{ env.SOURCE_REF_SANITIZED }}
+      #       ${{ steps.login-ecr.outputs.registry }}/dsva/preview-environment/${{ env.SOURCE_REPO }}:${{ env.SOURCE_REF_SANITIZED }}
```

Comment:
Is this supposed to be commented out?

Reply:
yes for the time being. We don't have the correct dockerfile built yet so until we do, we don't want to be pushing "test" images up to ECR.